zeppelin-interpreter/src/main/java/org/apache/zeppelin/conf/ZeppelinConfiguration.java

```diff
@@ -636,8 +633,12 @@ public String getInterpreterPortRange() {
     return getString(ConfVars.ZEPPELIN_INTERPRETER_RPC_PORTRANGE);
   }
 
-  public boolean isWindowsPath(String path){
-    return path.matches("^[A-Za-z]:\\\\.*");
+  public boolean isWindowsCheck(){
```

Comment:
I'm not sure but how about using `isWindows` or `checkWindows` as a method name?

Reply:
OK, I will do what you say.

Reply:
nitpick) How about `isWindowsOS`? It would be better to specify what to check.